src/api/app/models/project.rb

```diff
@@ -1750,6 +1753,15 @@ def has_local_distribution(project_name, repository)
         linked_repository.name == repository
     end
   end
+
+  def update_staging_workflow
+    transaction do
```

Comment:
This is still wrapped in the ``save`` transaction.
https://apidock.com/rails/ActiveRecord/Callbacks/after_save

Reply:
@ChrisBr thanks :smile_cat: